.github/workflows/build-assets.yml

```diff
@@ -251,14 +251,14 @@ jobs:
           builder: ${{ steps.buildx.outputs.name }}
           file: ${{ needs.preamble.outputs.folder_platform }}/dockerfiles/vcpkg/platform-core-${{ matrix.os }}/Dockerfile
           context: ${{ needs.preamble.outputs.folder_build }}
-          push: ${{ github.repository_owner == 'hpcc-systems' }}
+          push: true
```

Comment:
My concern here, was that testers (me) who were using DOCKER_USERNAME of `hpccsystems` pushing to dockerhub by mistake?